go.mod

```diff
@@ -4,11 +4,14 @@ go 1.19
 
 require (
 	github.com/Microsoft/go-winio v0.5.2
+	github.com/containerd/log v0.1.0
 	github.com/opencontainers/go-digest v1.0.0
-	github.com/sirupsen/logrus v1.8.1
-	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
-	golang.org/x/sys v0.1.0
+	golang.org/x/sync v0.1.0
+	golang.org/x/sys v0.7.0
 	google.golang.org/protobuf v1.33.0
 )
 
-require github.com/stretchr/testify v1.3.0 // indirect
+require (
+	github.com/sirupsen/logrus v1.9.3 // indirect
+	gopkg.in/yaml.v3 v3.0.1 // indirect
```

Comment:
👀

Reply:
containers/log doesn't seem to depend on yaml 🤔

Reply:
Yeah, it's through testify; 


```bash
go mod graph | grep ' gopkg.in/yaml.v3'
github.com/containerd/continuity gopkg.in/yaml.v3@v3.0.1
github.com/stretchr/testify@v1.7.0 gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c
```

Which is depended on through logrus;

```bash
go mod graph | grep ' github.com/stretchr/testify@v1.7.0'
github.com/sirupsen/logrus@v1.9.3 github.com/stretchr/testify@v1.7.0
```

Reply:
Ah! I think I know what happened; it can be removed indeed. The `v3.0.0-20200313102051-9f266ea9e77c` -> `v3.0.1` may have been either a manual bump (to prevent an old version with CVEs), or possibly stuck after it was updated indirectly. Go mod doesn't remove such lines automatically, but it doesn't add them back if you remove them manually; it's worth sometimes removing all the `// indirect`  dependencies, and see if there's ones that are _actually_ gone.

let me open a PR to get rid of it.